src/Overeno.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Heureka;
+
+/**
+ * @author vladimir.kaspar <vladimir.kaspar@heureka.cz>
+ */
+class Overeno
+{
+
+    /** Language IDs */
+    const LANGUAGE_CZ = 'cz';
+    const LANGUAGE_SK = 'sk';
+
+    /** Heureka API URL */
+    const URL_PATTERN = 'http://www.heureka.%s/direct/dotaznik/objednavka.php';
```

Comment:
Žádná ještě neexistuje:-D Něco na `api.` by bylo fajn. Akorát by to znamenalo víc úprav:(